src/MembershipsSDKAdapter.js

```diff
@@ -117,11 +136,17 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
       inMeeting: null,
     });
 
+    const currentUser$ = new PeopleSDKAdapter(this.datasource).getMe();
```

Comment:
I don't like the fact that a new instance of people's adapter is created. We should be able to use the same one. However, it's a "sibling" adapter, I'm not sure if there's currently any way to access it. @lalli-flores

Reply:
I'm thinking of adding a "link" to the `WebexSDKAdapter` instance from each specific adapter instance. This could be done by sending an extra parameter to each adapter constructor:
```js
this.peopleAdapter = new PeopleSDKAdapter(sdk, this);
```
https://github.com/webex/sdk-component-adapter/blob/master/src/WebexSDKAdapter.js#L16

Having this, we could easily access sibling adapters:
```js
const me$ = this.parent.peopleAdapter.getMe(); // or this.main, this.adapter, this.adapters, this.siblingAdapters, this.siblings, ...
```